README.md

```diff
@@ -4,23 +4,60 @@ A simple [ansible](http://docs.ansible.com) playbook to deploy a fullblown [geOr
 
 Right now, this will deploy a working geOrchestra from the **master** branch with mostly default configs and empty databases/datadirs.
 
+## Prerequisite
+
+* Debian Buster (10.x) VM
+* JAVA8 (could use [AdoptOpenJDK](https://adoptopenjdk.net/) project)
+* Token access to allow connection from VM to GitHub and allow to get MapStore2 artifact (see playbook.yml)
 
 ## setup
 
 In order to deploy all the middleware and components of a geOrchestra instance, you just need to:
+
  * use `ansible-galaxy` to install external roles required for geonetwork 4:
 ```
 ansible-galaxy install -r requirements.yaml
 ```
- * setup variables for your own instance in ```playbooks/georchestra.yml```,
- * point to the IP of a host where you have ssh-with-passphrase root access in the ```hosts``` file (ideally, an lxc container, or a vm, whatever suits you)
+
+ * Clone source :
+
+    `git clone https://github.com/georchestra/ansible.git`
+
+ * setup variables for your own instance in ```playbooks/georchestra.yml```
+
+ * Open `ansible/hosts` file
```

Comment:
why `ansible` subdir here ? at that point one can assume the admin went to the subdir to edit `playbooks/georchestra.yml`

Reply:
indeed... this was to show where is hosts file (inside ansible root folder).